orchagent/portsorch.cpp

```diff
@@ -8126,6 +8431,13 @@ std::unordered_set<std::string> PortsOrch::generateCounterStats(const string& ty
             counter_stats.emplace(sai_serialize_port_stat(it));
         }
     }
+    else if (type == WRED_PORT_STAT_COUNTER_FLEX_COUNTER_GROUP)
+    {
+        for (const auto& it: wred_port_stat_ids)
```

Comment:
Now that counter capabilities have been fetched from vendor SAI in [initCounterCapabilities](https://github.com/sonic-net/sonic-swss/pull/2750/files#diff-bb736c6a11da68afacf95d761fb52a3c11726cdd72c967d4803b8c9b1a6dcb69R912), why not just adding the supported counters?

Reply:
Flexcounter polling at syncd, polls only the supported statistics. So it is not required to filter it here.